### DISCUSSION TITLE
Sharpie vs. other markers/paints/inks

### DISCUSSION THREAD

**Discussion by dmarx** [Sakati84/3DPrintColorizer#21] [Ideas]
Sharpies clearly get the job done here, but they also probably [interfere with layer adhesion](https://forum.prusaprinters.org/forum/english-forum-general-discussion-announcements-and-releases/new-method-of-printing-perfect-supports/). Has anyone tried doing this with different kinds of markers or inks? Some ideas:

* fabric markers
* paint pens
* whiteboard markers
* chalkboard markers
* gel markers

I bet with the right combination of filament and ink/paint, this method could be competitive with true multi-material!

**Comment by Sakati84** [comment]
I tried Amazon and ShuttleArt markers. They didn‘t seem to be much different. Then again I tested them with my dialed in settings that don’t suffer from this problem too much
